api/src/main/java/org/zalando/nakadi/client/java/model/EventMetadata.java

```diff
@@ -77,6 +77,7 @@ public String getEid() {
         return eid;
     }
 
+    @JsonProperty("event_type")
```

Comment:
Yes it is necessary, the problem is that the teams are defining events without eventMetadata.
Because we tried to use it, that is how we bumped into this problem.